File: api/pipeline/health.ts
export default function handler(req: any, res: any) {
  if (req.method && req.method !== 'GET') {
    res.status(405).json({ error: 'Method Not Allowed' });
    return;
  }
  res.setHeader('Cache-Control', 'no-store');
  res.status(200).json({ status: 'ok' });
}

File: api/calculator/health.py
import json
import os
import sys

BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
CALC_DIR = os.path.join(BASE_DIR, 'services', 'calculator')
if CALC_DIR not in sys.path:
    sys.path.append(CALC_DIR)

from formulas import FORMULAS  # noqa: E402


def handler(request):
    if getattr(request, 'method', 'GET') != 'GET':
        return {
            "statusCode": 405,
            "headers": {"content-type": "application/json"},
            "body": json.dumps({"error": "Method Not Allowed"}),
        }

    payload = {"status": "ok", "formulas_available": list(FORMULAS.keys())}
    return {
        "statusCode": 200,
        "headers": {"content-type": "application/json"},
        "body": json.dumps(payload),
    }
